lighthouse/src/main.rs

```diff
@@ -84,7 +84,8 @@ fn main() {
         .arg(
             Arg::with_name("env_log")
                 .short("l")
-                .help("Enables environment logging giving access to sub-protocol logs such as discv5 and libp2p",
+                .help(
+                    "DEPRECATED Enables environment logging giving access to sub-protocol logs such as discv5 and libp2p",
```

Comment:
Looks like we're still using this flag here
https://github.com/sigp/lighthouse/pull/4979/files#diff-963f1419112461185d9fc5a49739e127c09755311f1a540ff39af2efe0329e89R379

Are we still deprecating this?

Reply:
Yeah. This is being deprecated.  See https://github.com/sigp/lighthouse/issues/4954